src/constants.ts

```diff
@@ -36,7 +36,7 @@ export const EXPAND_TYPE_DEFINITION = `type ExpandType<T> = unknown extends T
 \t? T extends unknown
 \t\t? { expand?: unknown }
 \t\t: { expand: T }
-\t: { expand: T }`
+\t: { expand?: T }`
```

Comment:
This is closer to reality - when there isn't any related items expand is `undefined`.